pkg/api/v1beta3/types.go

```diff
@@ -634,11 +634,13 @@ type Container struct {
 	// Optional: Defaults to /dev/termination-log
 	TerminationMessagePath string `json:"terminationMessagePath,omitempty" description:"path at which the file to which the container's termination message will be written is mounted into the container's filesystem; message written is intended to be brief final status, such as an assertion failure message; defaults to /dev/termination-log; cannot be updated"`
 	// Optional: Default to false.
-	Privileged bool `json:"privileged,omitempty" description:"whether or not the container is granted privileged status; defaults to false; cannot be updated"`
+	Privileged bool `json:"privileged,omitempty" description:"deprecated: whether or not the container is granted privileged status; defaults to false; cannot be updated  See SecurityContext."`
```

Comment:
Description ought to be `whether or not the container is granted privileged status; defaults to false; deprecated; cannot be updated; see SecurityContext`